src/styles/blocks/header.scss

```diff
@@ -0,0 +1,55 @@
+
+.header {
+  font-size: 12px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 2px 4px rgba(0, 0, 0, 0.05);
+  padding: 0 50px;
+
+  &__link {
+    display: flex;
+  }
+
+}
+
+.nav {
+
+  &__list {
+    list-style: none;
+    display: flex;
+    padding: 0;
+  }
+
+  &__link {
+    text-decoration: none;
+    color: #000;
+    padding: 23px 0;
+    cursor: pointer;
+    text-transform: uppercase;
+    position: relative;
+    line-height: 37px;
+
+    &:hover {
+      color: #00acdc;
+    }
+  }
+}
+
+.nav__list > li:not(:last-child) {
```

Comment:
I believe that you can avoid increasing selectors specificity. And get used to style all elements using classes